app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
In general, for all text styles you can create an XML file like in the example:
https://github.com/eminuluyol/MovieDBKotlin/blob/master/core/src/main/res/values/styles_textapearance.xml

and for the UI elements, you can create styles for the ones that are reusable